cirq-core/cirq/transformers/target_gatesets/compilation_target_gateset.py

```diff
@@ -115,3 +115,95 @@ def postprocess_transformers(self) -> List['cirq.TRANSFORMER']:
             drop_negligible.drop_negligible_operations,
             drop_empty_moments.drop_empty_moments,
         ]
+
+
+class TwoQubitCompilationTargetGateset(CompilationTargetGateset):
```

Comment:
Why do we need this interface ? 

If we look at `CZGateset` it doesn't seem to gain much by inheriting from this vs just inheriting straight from `CompilationTargetGateset`.

Reply:
Cleared up in offline chat. Would still be good to shorten name here if possible.

Reply:
Since the name will only be used while defining new classes that inherit from this class, I think we can let it be a a bit verbose in favour of having more clarity.